docs/changes/utils/13052.feature.rst

```diff
@@ -0,0 +1,9 @@
+Improve handling of unavailable IERS-B (predictive future Earth rotation) data
```

Comment:
Isn't it A? I thought B is the backup? Maybe I have confused myself very much. 😅 

```suggestion
Improve handling of unavailable IERS-A (predictive future Earth rotation) data
```

Reply:
Yes, and this A/B is always confusing to me even when I'm in the middle of it!